sdk/core/core-amqp/src/errors.ts

```diff
@@ -435,38 +435,94 @@ export enum ErrorNameConditionMapper {
   SystemError = "system:error"
 }
 
+/**
+ * Describes the fields on a Node.js SystemError.
+ * Omits fields that are not related to network calls (e.g. file system calls).
+ * See https://nodejs.org/dist/latest-v12.x/docs/api/errors.html#errors_class_systemerror
+ */
+interface NetworkSystemError {
+  address?: string;
+  code: string;
+  errno: string | number;
+  info?: any;
+  message: string;
+  name: string;
+  port?: number;
+  stack: string;
+  syscall: string;
+}
+
+const systemErrorFieldsToCopy: (keyof Omit<NetworkSystemError, "name" | "message">)[] = [
+  "address",
+  "code",
+  "errno",
+  "info",
+  "port",
+  "stack",
+  "syscall"
+];
+
 /**
  * Describes the base class for Messaging Error.
  * @class {MessagingError}
  * @extends Error
  */
 export class MessagingError extends Error {
   /**
-   * @property {string} [condition] The error condition.
+   * Address to which the network connection failed.
+   * Only available in Node.js.
```

Comment:
For properties that are applicable only for the System Errors, add a note hinting the same rather than `Only available in Node.js`?

Reply:
Updated to:
> Only present if the `MessagingError` was instantiated with a Node.js `SystemError`.